pkg/storage/common/common.go

```diff
@@ -80,7 +83,7 @@ func ValidateManifest(imgStore storageTypes.ImageStore, repo, reference, mediaTy
 		if err := ValidateManifestSchema(body); err != nil {
 			log.Error().Err(err).Msg("OCIv1 image manifest schema validation failed")
 
-			return "", zerr.ErrBadManifest
+			return "", zerr.ErrWrapKey(zerr.ErrBadManifest, SchemaErrKey, err.Error())
```

Comment:
How about using 
```suggestion
			return "", fmt.Errorf(%w (%w)", zerr.ErrBadManifest, err)
```

Reply:
This can work, although not extensible (like custom wrap and unwrap on demand).